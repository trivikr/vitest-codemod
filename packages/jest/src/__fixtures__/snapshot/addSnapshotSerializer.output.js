import { describe, expect, test } from "vitest";
describe("addSnapshotSerializer", () => {
  expect.addSnapshotSerializer({
    serialize(val) {
      return JSON.stringify(val);
    },
    test(val) {
      return val;
    }
  });

  describe("toMatchSnapshot", () => {
    test("string", () => {
      expect("foobar".toUpperCase()).toMatchSnapshot();
    });

    test("object", () => {
      expect({ foo: "bar" }).toMatchSnapshot();
    });
  });

  describe("toMatchInlineSnapshot", () => {
    test("string", () => {
      expect("foobar".toUpperCase()).toMatchInlineSnapshot(`"FOOBAR"`);
    });

    test("object", () => {
      expect({ foo: "bar" }).toMatchInlineSnapshot(`{"foo":"bar"}`);
    });
  });
});