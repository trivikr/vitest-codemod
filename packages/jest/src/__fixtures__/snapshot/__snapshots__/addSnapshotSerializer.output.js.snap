// Vitest Snapshot v1

exports[`addSnapshotSerializer > toMatchSnapshot > object 1`] = `{"foo":"bar"}`;

exports[`addSnapshotSerializer > toMatchSnapshot > string 1`] = `"FOOBAR"`;
